File: programmers/etc/level1/둘만의_암호.py
def solution(s, skip, index):
    answer = ""
    alpha = "abcdefghijklmnopqrstuvwxyz"
    for i in skip:
        alpha = alpha.replace(i, "")
    for i in s:
        answer += alpha[(alpha.find(i)+index)%len(alpha)]
    return answer


s, skip = "aukks", "wbqd"
index = 5
print(solution(s, skip, index)) # "happy"

File: Java/programmers/etc/level1/둘만의암호.java
package CodeTest.Java.programmers.etc.level1;

public class 둘만의암호 {
    public String solution(String st, String skip, int index) {
        String answer = "";
        String alpha = "abcdefghijklmnopqrstuvwxyz";

        for (String i : skip.split("")) {
            alpha = alpha.replaceAll(i, "");
        }

        int n = alpha.length();
        for (String s : st.split("")) {
            int p = (alpha.indexOf(s)+index)%n;
            answer += alpha.substring(p, p+1);
        }
 
        return answer;
    }
}
